converters/toZigbee.js

```diff
@@ -2477,7 +2477,7 @@ const converters = {
                     await entity.command('closuresWindowCovering', 'stop', {}, utils.getOptions(meta.mapped, entity));
                 }
 
```

Comment:
We also need to add the following code here:

```javascript

                return {state: {state: 'STOP'}};

```

Reply:
After "await entity.command..." or after the close bracket ?

Reply:
On line _2479_ (below the close bracket):

```javascript
                    await entity.command('closuresWindowCovering', 'stop', {}, utils.getOptions(meta.mapped, entity));
                }

                return {state: {state: 'STOP'}};

                if (!['ZNCLDJ11LM', 'ZNJLBL01LM', 'ZNCLBL01LM'].includes(meta.mapped.model)) {
```

Reply:
if I add the return statement there, wouldn't it skip the if statement below?

Reply:
Yes, thanks, we need to add it even lower:

```javascript
                    await entity.read('genAnalogOutput', [0x0055]);
                }

                return {state: {state: 'STOP'}};
            } else {
                const lookup = {'open': 100, 'close': 0, 'on': 100, 'off': 0};
```